src/plugins/shared_ux/public/components/toolbar/solution_toolbar/quick_group_buttons/quick_group.tsx

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import React from 'react';
+import {
+  EuiButtonGroup,
+  htmlIdGenerator,
+  EuiButtonGroupOptionProps,
+  useEuiTheme,
+  IconType,
+} from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import { css } from '@emotion/react';
+
+const strings = {
+  getAriaButtonLabel: (createType: string) =>
+    i18n.translate('sharedUX.solutionToolbar.quickButton.ariaButtonLabel', {
+      defaultMessage: `Create new {createType}`,
```

Comment:
This is a carry-over from Dashboard, as well... it restricts the quick buttons to "creating something".  I don't think that's a limitation for us.

I would get rid of this, and instead, offer `label` and `title` on the `QuickButtonProps` interface.  You could make `title` optional and default it to `label`.  Dashboard and Canvas would then still keep this function in their codebases to populate `title` differently.

Reply:
I think I'm following - a014375 for changes thanks!